### PR TITLE
Add talks for the winter semester 2017/2018

### PR DESCRIPTION
**Pull request by neithernut**
Please give some feedback:
@asuivelentine
@manuelmessner
@Senkor

### CODE REVIEW

File: content/talks/2017-12-14-PEARL.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+title: Fuwa News Show
```
Comment (matthiasbeyer) [inline content/talks/2017-12-14-PEARL.md]: Copy-Paste fail.

File: content/talks/2017-12-21-Challenge.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+---
+title: Fuwa News Show
```
Comment (matthiasbeyer) [inline content/talks/2017-12-21-Challenge.md]: Copy-Paste fail.

**Review by matthiasbeyer — CHANGES_REQUESTED** [review]
Wow, all dates are taken! Looks awesome, I'd love to attend some dates.

**Comment by asuivelentine** [comment]
with the changes mentioned by @matthiasbeyer this looks good for me.

**Comment by neithernut** [comment]
Any other suggestions? Could someone merge this, please?

**Comment by matthiasbeyer** [comment]
IMO ack.

**Pull request merged**